docs/contributing.md

```diff
@@ -1,30 +1,30 @@
 # Contributing to Application Services
 
-Anyone is welcome to help with the Application Services project. Feel free to get in touch with other community members on Matrix, the mailing list or through issues here on GitHub.
+Anyone is welcome to help with the [Application Services](README.md) project. Feel free to get in touch with other community members on Matrix, the mailing list or through issues here on GitHub.
```

Comment:
I noticed you removed "here" below, which kinda makes sense, so I wonder if it should be removed here too?